applications/ParticleMechanicsApplication/particle_mechanics_application_variables.h

```diff
@@ -19,6 +19,7 @@
 #include "includes/define.h"
 #include "includes/constitutive_law.h"
 #include "includes/variables.h"
+#include "includes/cfd_variables.h"
```

Comment:
```suggestion
```

please do not include this globally, it increases the compilation time
=> it should only be included when you actually need it

Reply:
Thanks @philbucher. I use it in the constitutive law and the UP element. Adding it just there is ok? Is it better in the .hpp or the .cpp?

Reply:
Yes only in the files where you need it. if possible only in the cpp. If you also need it in the hpp then only there (since the hpp gets included in the cpp)

Reply:
Thank you. Sorry, but now I see that not all modifications I made were reported. Is it ok if I close the pull request and re-open when it will be right? Sorry, my fault, but I am new to this